src/components/UserManagement/logTimeOffPopUp.jsx

```diff
@@ -4,7 +4,6 @@ import moment from 'moment-timezone';
 import {
   Button,
   Modal,
-  ModalHeader,
```

Comment:
This error was caused by the deletion of this import.
![Screenshot 2024-05-11 230729](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49083865/404311cd-b438-41e7-9622-e330181a2707)